marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/MergeDatabaseConfigTest.java

```diff
@@ -0,0 +1,89 @@
+package com.marklogic.hub.central.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MergeDatabaseConfigTest {
+
+    @Test
+    public void testMergeIndexConfigs() throws JsonProcessingException {
```

Comment:
I think you also need a scenario for where the entity indexes already exist. So keep this scenario, and add another one that has the same setup here, and the existing indexes includes the entity indexes.

Reply:
Added the scenario in here da196c8